dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventUtils.java

```diff
@@ -50,4 +64,136 @@ public static UserInfoSnapshot jsonToUserInfo(String userInfoAsString, ObjectMap
       throw new IllegalArgumentException(e);
     }
   }
+
+  public static List<OrganisationUnit> validateAccessibleOrgUnits(
```

Comment:
I don't want us to make concessions in new tracker code due to changes needed for old tracker 😅 Not a fan of utils 😂 

Can we either
* duplicate the code (which is our strategy so far) so that we stay independent
* move this in an existing ACL related class?

Reply:
I agree, I would leave this code in `EventOperationParamsMapper` and create a new `EventUtils` or something similar, in the deprecated tracker